### ISSUE TITLE
Custom fields support from PODS

### ISSUE THREAD

**Issue by looknear** [Crocoblock/suggestions#1523]
ACF is supported, but PODS need to be supported too.

PODS is an open source powerful custom fields and custom post plugin with very stable architecture and solid code base and community. 
Elementor is also supporting all PODS fields.
Will be great to support all PODS features across your plugins. With Dynamic Fields etc.

**Comment by secondsky** [comment]
Indeed! Should be not that difficult to implement. A look to other plugins and how they do it should help...
